podcasts/Whats New/WhatsNewView.swift

```diff
@@ -0,0 +1,53 @@
+import SwiftUI
+
+struct WhatsNewView: View {
+    @EnvironmentObject var theme: Theme
+
+    let announcement: WhatsNewAnnouncement
+
+    var body: some View {
+        VStack(spacing: 10) {
+            ZStack(alignment: .top) {
+                Rectangle()
+                    .frame(height: 195)
+
+                HStack {
+                    Spacer()
+                    Button {
+                        NavigationManager.sharedManager.dismissPresentedViewController()
+                    } label: {
+                        ZStack {
+                            Image("close")
+                                .foregroundColor(.white)
```

Comment:
We should switch to using `foregroundStyle` because `foregroundColor` is [now deprecated](https://developer.apple.com/documentation/swiftui/view/foregroundcolor(_:)).

```suggestion
                                .foregroundStyle(.white)
```